Jenkinsfile

```diff
@@ -67,22 +67,7 @@ def generatePackageSteps(opts) {
     }
 }
 
-def packageBuildSteps = [
-    "windows": { ->
-        node("windows-2022") {
-            stage("windows") {
-                try {
-                    checkout scm
-                    sh("make -f Makefile.win archive")
-                } finally {
-                    deleteDir()
-                }
```

Comment:
Should we catch the error, so that we still have information about the failures while we're investigating? Not exactly sure what the right format is, but this SO post has some examples
https://stackoverflow.com/questions/44022775/jenkins-ignore-failure-in-pipeline-build-step

Reply:
Hmmm good one, will try it out